task_12/style/sass/buttons.sass

```diff
@@ -0,0 +1,59 @@
+.button 
+    padding: 1em 2em 
+    text-transform: uppercase 
+    font-weight: 700
+    background: none
+    border: none
+    cursor: pointer
+    box-shadow: 0 1px 3px rgba(0,0,0,0.12), 0 1px 2px rgba(0,0,0,0.24)
+    transition: all 0.3s cubic-bezier(.25,.8,.25,1)
+    position: relative
+    i 
+        margin: 0 0.5em 0 0
+    &:hover
+        box-shadow: 0 14px 28px rgba(0,0,0,0.25), 0 10px 10px rgba(0,0,0,0.22)
+    &__dark-orange 
```

Comment:
It looks more like modifier than child element. Read more about BEM.

Reply:
Read thread about BEM [here](https://github.com/kottans-frontend-2018/assignments_yevhenorlov/pull/1#pullrequestreview-92063523) (click _show outdated_ to view)

Reply:
Thanks a lot for information about BEM! It seems that i'm used to use incorrect class namings. I rearranged styles a bit.